watertap/examples/flowsheets/full_treatment_train/flowsheet_components/chemistry/posttreatment_ideal_naocl_chlorination_block.py

```diff
@@ -695,14 +680,10 @@ def run_ideal_naocl_mixer_example(fixed_dosage=False):
 
     check_dof(model)
 
-    # scale the mixer
-    scale_ideal_naocl_mixer(model.fs.ideal_naocl_mixer_unit)
```

Comment:
Why omit scaling of the mixer?

Reply:
Ah, because it just uses `iscale.constraint_autoscale_large_jac()` which is being eliminated from all code in this PR.